packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/AndroidWebkitLibrary.g.kt

```diff
@@ -86,7 +86,8 @@ class AndroidWebkitLibraryPigeonInstanceManager(
    */
   var clearFinalizedWeakReferencesInterval: Long = 3000
     set(value) {
-      handler.removeCallbacks { this.releaseAllFinalizedInstances() }
+      val runnable = Runnable { this.releaseAllFinalizedInstances() }
```

Comment:
This is bypassing the warning rather than fixing what the warning is flagging, which is that this will not remove the listener added on line 96.

The goal with warnings is to address the cause of the warning, rather than to silence the warning without fixing the underlying problem.